File: iotcloud-restapi-java/Core/src/com/qcloud/restapi/iotcore/response/CreateProductResponse.java
package com.qcloud.restapi.iotcore.response;

import com.google.gson.*;

public class CreateProductResponse extends Response {
    /**
     * 解析JSON字串并构造CreateProductResponse对象
     *
     * @param response 从服务端返回的应答JSON字串
     */
    public CreateProductResponse(String response) {
        super(response);
    }

    public String getProductID() {
        try {
            if (jsonObj != null) {
                return jsonObj.get("productID").getAsString();
            }
        }catch (Exception e) {
        }

        return "";
    }

    public String getProductName() {
        try {
            if (jsonObj != null) {
                return jsonObj.get("productName").getAsString();
            }
        }catch (Exception e) {
        }

        return "";
    }
}

File: iotcloud-restapi-php/Core/src/IoTCloud/Response/CreateProductResponse.php
<?php

namespace TXIoTCloud\Response;

require_once "BaseResponse.php";

class CreateProductResponse extends BaseResponse
{
    /**
     * 产品名称
     */
    private $productName;

    /**
     * 产品ID
     */
    private $productID;

    public function __construct()
    {
    }

    /**
     * 获取产品名称
     */
    public function getProductName()
    {
        return $this->productName;
    }

    /**
     * 设置产品名称
     * @param
     */
    public function setProductName($productName)
    {
        $this->productName = $productName;
    }

    /**
     * 获取产品ID
     */
    public function getProductId()
    {
        return $this->productID;
    }

    /**
     * 设置产品ID
     * @param
     */
    public function setProductId($productID)
    {
        $this->productID = $productID;
    }


}